File: DAY 1/JAVA/Guess.java

import java.util.Scanner;
import java.util.Random;
public class Guess{
public static void main(String [] args){

        Scanner input = new Scanner(System.in);
        Random random = new Random();
        int guess = random.nextInt(100) +1;    
        int numberofAttempt = 0;
       System.out.println("Enter a Number: ");
       int number = input.nextInt();
   
        for(int count = 0; count <= 5; count++){
     
    System.out.print("Enter Number between 1 and 100: ");
        int userinput = input.nextInt();
      if(userinput <= -1){
      System.out.println("Do not count");
      }  
      else if(userinput > number){
      System.out.println("Answer is Higher");
      } 
      else if(userinput < number){
      System.out.println("Answer is lower");
      }
      else if(userinput == number){
        
         System.out.println("Stop");
        break;
      } 
      if(guess == 1){
      System.out.println("Legendary");
      }
      else if(guess == 2){
      System.out.println("Excellent");
      
      }
      else if(guess == 3 && guess == 4){
      System.out.println("Good");
      }
      else if(guess == 5){
      System.out.println("Close");
        }
        else{
        System.out.println("Better luck");
}
}

}

}

File: DAY 1/PYTHON/guess.py

import random
wrong = random.randInt(1, 100) + 1
guesses = 0
print("Guess my number between 1 and 100 :")
while guess < 5:  
        guess_number = int(input("Enter your guess: "))
        
        
        if guess_number < 1 or guess_number > 100:
            print("1 - 100 only")
            continue 
guesses += 1  
if guess_number == wrong:
   print("You win")
break
   
elif guess_number > wrong:
   print("Too high. Try again.")
else:  
   print("Too low. Try again.")
   break    
  
        
if(guess == 1):
   print("Lengendary")
                
elif(guess == 2):
    print("Excellent")
elif(guess == 3 and guess == 4):
    print("Good")
elif (guess == 5):
    print("Close")
else:
    print("Better luck")
                
          
    
    
    
    
       
